README.md

```diff
@@ -91,6 +98,23 @@ else fails, try removing the virtual environment and reinstalling:
 
 [local5000]:http://localhost:5000/
 
+### Troubleshooting Possible Errors
+
+* Should you get series of type errors that looks something like `npm ERR! typeerror Error: Missing required argument #1`, after running `pipenv run lektor server -f webpack`, this is most likely due to running an older version of Node.js 
+
+As stated earlier in the prerequisites, you should be running Node.js versions 12+.
+
+Follow [this](https://github.com/nodesource/distributions/blob/master/README.md#installation-instructions) tutorial to upgrade your node version (for GNU/Unix systems).
```

Comment:
A small suggestion. To indent the paragraphs after a bullet point add two spaces before the first line of the paragraph. This will visually indicate that the paragraphs are in the same context as the bullet point.

```suggestion
* Should you get series of type errors that looks something like `npm ERR! typeerror Error: Missing required argument #1`, after running `pipenv run lektor server -f webpack`, this is most likely due to running an older version of Node.js 

  As stated earlier in the prerequisites, you should be running Node.js versions 12+.

  Follow [this](https://github.com/nodesource/distributions/blob/master/README.md#installation-instructions) tutorial to upgrade your node version (for GNU/Unix systems).
```

Reply:
Issues addressed  :)

Reply:
Please do the same for the next bullet point also.